packages/objects/src/interfaces/IConsentBaseURISchema.ts

```diff
@@ -0,0 +1,8 @@
+import { IUserAgreement } from "@objects/interfaces/IUserAgreement.js";
+import { URLString } from "@objects/primitives/URLString.js";
+
+export interface IConsentBaseURISchema extends IUserAgreement {
+  brandName?: string;
```

Comment:
Instead of defining three separate fields (assuming they are optional to support older versions), we can consolidate them into a single field that encompasses all three, or leave it optional.